### ISSUE TITLE
Show oldest or latest unread toot when opening app

### ISSUE THREAD

**Issue by mase76** [stom79/mastalab#472]
Hi!
When opening the app, it should show either the oldest or the latest unread toot. This could be configurable. The current behaviour is nearly undefined. It starts with a toot somewhere in the upper part of the timeline. Sometimes an already read toot is shown as new, also toots I wrote.

**Comment by stom79** [comment]
Currently the app automatically remembers your last toot read in the timeline see #34. If a toot has been read (ie displayed), it should not have the tag new. I didn't encounter this issue right now. Could you bring more details:
- Do you use multiple accounts?
- If yes, do these toots were read by the same account?

**Comment by mase76** [comment]
No, I use only one account. Most times after opening, the last (read) toot is reloaded. The last shown position is not shown every time, especially when I closed the app, being on top of the timeline. The new tag in old toots does not appear frequent. Sometimes I see a notification dot with a number in the tap bar, but there is nothing new.

**Comment by mase76** [comment]
No, I use only one account. Most times after opening, the last (read) toot is reloaded. The last shown position is not shown every time, especially when I closed the app, being on top of the timeline. The new tag in old toots does not appear frequent. Sometimes I see a notification dot with a number in the tap bar, but there is  nothing new.
